Options.py

```diff
@@ -854,13 +855,51 @@ def __iter__(self) -> typing.Iterator[str]:
     def __len__(self) -> int:
         return self.value.__len__()
 
+    # __getitem__ fallback fails for Counters, so we define this explicitly
+    def __contains__(self, item):
+        return item in self.value
+
+
+class CounterOption(OptionDict):
+    min: typing.Optional[int] = None
```

Comment:
Debatable but I think a default min of 1 probably has the most universal use cases

Reply:
I'm not sure about that? I'm aware of three options that use this or would want to use this, and two of them want min = 0.